docs/getting-started/upgrade-to-v4.md

```diff
@@ -63,6 +63,92 @@ The Adapter API has been rewritten and significantly simplified in NextAuth v4.
 
 **This does not require any changes from the user**, however if you are an adapter maintainer or are interested in writing your own adapter, you can find more information about this change in https://github.com/nextauthjs/next-auth/pull/2361 and release https://github.com/nextauthjs/next-auth/releases/tag/v4.0.0-next.22.
 
+### Schema changes
+
+The way we save data with adapters have slightly changed. With the new Adapter API, we wanted to make it easier to extend your database with additional fields. For example if your User needs an extra `phone` field, it should be enough to add that to your database's schema, and no changes will be necessary in your adapter.
+
+- `created_at`/`createdAt` and `updated_at`/`updatedAt` fields are removed from all Models.
+- `user_id`/`userId` consistently named `userId`.
+- `compound_id`/`compundId` is removed from Account.
+- `access_token`/`accessToken` is removed from Session.
+- `email_verified`/`emailVerified` on User is consistently named `email_verified`.
+- `provider_id`/`providerId` renamed to `provider` on Account
+- `provider_type`/`providerType` renamed to `type` on Account
+- `provider_account_id`/`providerAccountId` on Account is consistently named `providerAccountId`
+- `access_token_expires`/`accessTokenExpires` on Account renamed to `expires_in`
+- New fields on Account: `expires_at`, `token_type`, `scope`, `id_token`, `oauth_token_secret`, `oauth_token`, `session_state`
+  
+
+<!-- REVIEW: Would something like this below be helpful? -->
+<details>
+<summary>
+See the changes
+</summary>
+<pre>
+
+```diff
+User {
```

Comment:
This is a great start! It would make it even easier to get started if we could provide a SQL command, similar to v3 where one could simply run it to create all the tables without worrying about human error (and also getting started much quicker).

Reply:
Remember, SQL is only a tiny portion of what we support. I don't know if it's feasible for us to create migrations for everyone, but I welcome PRs/volunteers who would help us for the different adapters. 😊

Reply:
Once I get the SQL schema working, I can share it here. 🙌

Reply:
We have the migrations in our tests usually, maybe we can surface that for the adapters which don't automatically do the initial seeding/migrstion